printable-recipe-generator/src/main.ts

```diff
@@ -16,8 +71,15 @@ function renderJsonToHtml(recipeDataPath: string) {
 
 	//Render html
 	let renderHtml = '';
+	// @ts-expect-error -- we know this
```

Comment:
again, hopefully should not be necessary

Reply:
yeah